tangelo/linq/translator/translate_pennylane.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions helping with quantum circuit format conversion between abstract
+format and pennylane format.
+
+In order to produce an equivalent circuit for the target backend, it is
+necessary to account for:
+- how the gate names differ between the source backend to the target backend.
+- how the order and conventions for some of the inputs to the gate operations
+    may also differ.
+"""
+
+from math import pi
+
+from tangelo.helpers import deprecated
+
+
+def get_pennylane_gates():
+    """Map gate name of the abstract format to the equivalent methods of the
+    cirq class API and supported gates: https://quantumai.google/pennylane/gates.
```

Comment:
This says cirq, and the link does not work for me. I'm not sure what's going on here.